support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListContainer.tsx

```diff
@@ -21,12 +21,34 @@ type CheckoutBenefitsListContainerProps = {
 	renderBenefitsList: (props: CheckoutBenefitsListProps) => JSX.Element;
 };
 
-function getBenefitsListTitle(
+function getEmotionalBenefitsTitle(
 	priceString: string,
 	contributionType: ContributionType,
+	displayEmotionalBenefit: boolean,
 ) {
 	const billingPeriod = contributionType === 'MONTHLY' ? 'month' : 'year';
-	return `For ${priceString} per ${billingPeriod}, you’ll unlock`;
+	const emotionalMessage = displayEmotionalBenefit
+		? getEmotionalBenefit(priceString)
+		: ``;
+	return [
+		{ copy: `For ${priceString} per ${billingPeriod}`, strong: true },
+		`, `,
+		emotionalMessage,
+		`you’ll unlock`,
+	];
+}
+
+function getEmotionalBenefit(priceString: string) {
+	switch (priceString) {
+		case '$35':
+		case '$378':
+			return 'make a greater impact on the future of independent journalism and ';
+		case '$13':
+		case '$120':
+			return 'deepen your commitment to the Guardian’s independence and ';
+		default:
+			return `support access to independent journalism for all those who want and need it, `;
```

Comment:
Does this config need to consider the billing period as well (Monthly or Annual) or will the copy be the same regardless?

Reply:
Yep copy is the same for annual and monthly at these price cards